jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/complex/ProgressSectionHeightTests.java

```diff
@@ -0,0 +1,50 @@
+package io.github.epam.bootstrap.tests.complex;
+
+import com.epam.jdi.light.ui.bootstrap.elements.common.Progress;
+import com.epam.jdi.light.ui.bootstrap.elements.complex.ProgressBarWrapper;
+import io.github.epam.TestsInit;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import java.util.List;
+
+import static io.github.com.StaticSite.bsPage;
+import static io.github.com.pages.BootstrapPage.progressHeightSection;
+import static io.github.epam.states.States.shouldBeLoggedIn;
+import static org.testng.Assert.assertEquals;
+
+public class ProgressSectionHeightTests extends TestsInit {
+
+    public List<ProgressBarWrapper> list;
+
+    @BeforeMethod
+    public void before() {
+        shouldBeLoggedIn();
+        bsPage.shouldBeOpened();
+        list = progressHeightSection.getProgressBarWrappersList();
+    }
+
+    @Test
+    public void heightOfWrapperShouldBeValid() {
+        for (ProgressBarWrapper wrapper : list) {
+            int actualHeight = wrapper.core().getSize().getHeight();
+            int expectedHeight = wrapper.getWrapperHeightValueInPx();
+            assertEquals(actualHeight, expectedHeight);
+        }
+    }
+
+    @Test
+    public void heightOfBarShouldBeValid() {
+        for (ProgressBarWrapper wrapper : list) {
+            int expectedBarHeight = wrapper.getWrapperHeightValueInPx();
+
+            List<Progress> progressBarList = wrapper.getProgressBarList();
+            for (Progress progressBar : progressBarList) {
+                int actualBarHeight = progressBar.core().getSize().getHeight();
+                assertEquals(actualBarHeight, expectedBarHeight);
```

Comment:
could you please check , do we have already jdi asserts for width and height instead of testng? if not, we should create it for Progress

Reply:
fixed